lib/SIL/IR/OperandOwnership.cpp

```diff
@@ -913,6 +913,17 @@ OwnershipConstraintClassifier::visitBuiltinInst(BuiltinInst *bi) {
 Optional<OwnershipConstraint> Operand::getOwnershipConstraint() const {
   if (isTypeDependent())
     return None;
+
+  // If we do not have ownership enabled, just return any. This ensures that we
+  // do not have any consuming uses and everything from an ownership perspective
+  // is just a liveness use short-circuiting many of the optimizations.
+  //
+  // We do not ever call this function when an instruction isn't in a block.
+  assert(getUser()->getParent() &&
+         "Can not lookup ownership constraint unless inserted into block");
+  if (!getUser()->getFunction()->hasOwnership())
```

Comment:
But instructions are not always in a SILFunction!

Reply:
Touche. I did it right below. Wasn't thinking!